copier.yml

```diff
@@ -352,17 +348,19 @@ cidr_whitelist:
     ⚠️ It must be a list. And this is only supported if you deploy with Traefik 2+.
 
 postgres_version:
-  default: "14"
+  default: "15"
   help: >-
-    Which PostgreSQL version do you want to deploy?
+    Which PostgreSQL version do you want to deploy? (Recommended: for new instances
+    always use the lastest version)
   choices:
     I will use an external PostgreSQL server: ""
-    "9.6": "9.6"
-    "10": "10"
+    "9.6 (Minimum version for Odoo >=7.0)": "9.6"
+    "10 (Minimum version for Odoo >=14.0)": "10"
     "11": "11"
-    "12": "12"
+    "12 (Minimum version for Odoo >=16.0)": "12"
     "13": "13"
-    "14": "14"
+    "14 (Maximum version for Odoo <=12.0)": "14"
+    "15": "15"
```

Comment:
You can use a validator if you prefer ([new feature in Copier 6.2.0](https://github.com/copier-org/copier/blob/master/CHANGELOG.md#v620-2022-09-18)).

Reply:
I don't know what I'm doing wrong, but I can't get the "validator" to work... :/